skbio/stats/composition.py

```diff
@@ -1793,41 +1793,41 @@ def dirmult_ttest(
     pd.DataFrame
         A table of features, their log-fold changes and other relevant statistics.
 
-        `"T statistic"` is the t-statistic outputted from the t-test. T-statistics
-        are generated from each posterior draw.  The reported `T statistic` is the
+        ``t-statistic`` is the *t*-statistic outputted from the *t*-test. *t*-statistics
+        are generated from each posterior draw.  The reported ``T statistic`` is the
         average across all of the posterior draws.
 
-        `"df"` is the degrees of freedom from the t-test.
+        ``df`` is the degrees of freedom from the *t*-test.
 
-        `"Log2(FC)"` is the expected log2-fold change. Within each posterior draw
+        ``Log2(FC)`` is the expected log2-fold change. Within each posterior draw
         the log2 fold-change is computed as the difference between the mean
-        log-abundance the `treatment` group and the `reference` group. All log2
-        fold changes are expressed in clr coordinates. The reported `Log2(FC)`
+        log-abundance the ``treatment`` group and the ``reference`` group. All log2
+        fold changes are expressed in clr coordinates. The reported ``Log2(FC)``
         is the average of all of the log2-fold changes computed from each of the
         posterior draws.
 
-        `"CI(2.5)"` is the 2.5% quantile of the log2-fold change. The reported
-        `CI(2.5)` is the 2.5% quantile of all of the log2-fold changes computed
+        ``CI(2.5)`` is the 2.5% quantile of the log2-fold change. The reported
+        ``CI(2.5)`` is the 2.5% quantile of all of the log2-fold changes computed
         from each of the posterior draws.
 
-        `"CI(97.5)"` is the 97.5% quantile of the log2-fold change. The
-        reported `CI(97.5)` is the 97.5% quantile of all of the log2-fold
+        ``CI(97.5)`` is the 97.5% quantile of the log2-fold change. The
+        reported ``CI(97.5)`` is the 97.5% quantile of all of the log2-fold
         changes computed from each of the posterior draws.
 
-        `"pvalue`" is the pvalue of the t-test. The reported `pvalue` is the
+        ``p-value`` is the pvalue of the *t*-test. The reported ``*p*-value`` is the
```

Comment:
`pvalue` here refers to the variable name which the function returns, so it should be left as ``pvalue`` with two tick marks (`) on either side, with no hyphen or italics. Same with `qvalue` below.